gradle/dependencies.gradle

```diff
@@ -123,7 +123,7 @@ versions += [
   scoverage: "2.0.11",
   slf4j: "1.7.36",
   snappy: "1.1.10.7",
-  spotbugs: "4.8.6",
+  spotbugs: "4.9.1",
```

Comment:
ditto. If there is a strong reason to use `4.9.1` instead of `4.9.3`, could you please add comments explaining it?